openshift-with-appstudio-test/e2e/types.go

```diff
@@ -18,7 +18,3 @@ type testArgs struct {
 	pipeline *tektonpipeline.Pipeline
 	run      *tektonpipeline.PipelineRun
 }
-
-func (ta *testArgs) Logf(msg string) {
-	ta.t.Logf("time: %s: %s", time.Now().String(), msg)
-}
```

Comment:
The testing logging formatter does stack frame depth calculation to print out where the logging is from. If its redirected that breaks.